test/helpers/xhr/PASSWORD_RESET_withMaxComplexity.js

```diff
@@ -0,0 +1,50 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
+    "stateToken": "testStateToken",
+    "expiresAt": "2015-07-17T02:33:34.874Z",
+    "status": "PASSWORD_RESET",
+    "_embedded": {
+      "policy": {
+        "complexity": {
+          "minLength": 8,
+          "minLowerCase": 1,
```

Comment:
Just checking - is it true that this will always be one? Currently, it looks like the code only expects one for minLowerCase, upperCase, etc.

\+ @srinivasanagandla-okta